object-construction-checker/tests/mustcall/MustCallChoiceImplWrong1.java

```diff
@@ -0,0 +1,24 @@
+// A simple test that the extra obligations that MustCallChoice imposes are
+// respected. This version gets it wrong by not assigning the MCC param
+// to a field.
+
+import org.checkerframework.checker.mustcall.qual.*;
+import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.checker.objectconstruction.qual.*;
+import java.io.*;
+
+public class MustCallChoiceImplWrong1 implements Closeable {
+
+    final @Owning Closeable foo;
+
+    // :: error: required.method.not.called
```

Comment:
I'm not sure what would be better in this case - the `Closeable` parameter is the object that's warned about, and it does go out of scope without being closed.

Reply:
Well given the MCC annotations we might want to give an error like you didn't wrap the parameter as expected, or something like that.  But it's fine for now

Reply:
Thanks, I now understand and agree it would be possible to write an error message for precisely this situation, but I think for now it's fine (as you said originally) because the point does get across.